addons/rose/addon/styles/hds/overrides.scss

```diff
@@ -30,13 +30,6 @@
   .hds-table__td p {
     color: var(--token-form-indicator-optional-color);
   }
-
-  // TODO: This is a temporary hack to fix an issue causing weird whitespace
-  //   and scrollbars from an accessible class in HDS. We have to use !important
-  //   as they already forced it with !important and we can't override it otherwise
-  .sr-only {
```

Comment:
Whoa this can't be deleted. Read the comment again. It's not for our usage of `sr-only` but a temporary fix for its usage in HDS.

Reply:
@ZedLi t is not used by any other component?

Reply:
It's used by many HDS components. This is an override for an issue we had in desktop client that hopefully we can remove in the future

Reply:
@ZedLi ahh i see it now, its part of the table